adapt/intent.py

```diff
@@ -203,6 +285,62 @@ def build(self):
         """
         Constructs an intent from the builder's specifications.
 
-        :return: an Intent instance.
+        Returns: an Intent instance.
         """
-        return Intent(self.name, self.requires, self.at_least_one, self.optional)
\ No newline at end of file
+        return Intent(
+            self.name,
+            self.requires,
+            self.at_least_one,
+            self.optional)
+
+
+""" For testing locally """
+if __name__ == "__main__":
```

Comment:
I'm not really a fan of appending these to the implementation files. Can we create an integration tests folder? I'm not sure what this is providing over the existing unit tests.